docs/manual/guides/local-installation/devilbox.de.md

```diff
@@ -165,3 +187,115 @@ Der Benutzer `root` mit leerem Passwort ist die Devilbox Standard Einstellung. D
 die [Konfiguration](https://devilbox.readthedocs.io/en/latest/support/faq.html#can-i-change-the-mysql-root-password) geändert werden. 
 In diesem Fall musst du im Contao-Installtool deine Werte entsprechend eintragen.
 {{% /notice %}}
+
+
+## Nützliche Informationen: Mehrere php Versionen parallel betreiben {{< ab devilbox version "3" >}} 
+
+Die Grundeinstellung nimmst du im devilbox Verzeichnis vor.
+Im Verzeichnis `compose` findest du die Datei `docker-compose.override.yml-php-multi.yml`. Kopiere die Datei in das devilbox Hautpverzeichnis und ändere den Dateinamen nach »docker-compose.override.yml«. 
+
+Du hast ein Projekt <project>, das abweichend von der in der .env eingestellten php-Version (in unserem Beispiel 8.2) mit einer anderen php-Version - sagen wir 7.4 - laufen soll?
+Lege in diesem Projektverzeichnis ein Verzeichnis `.devilbox` und darin eine Datei `backend.cfg` an mit diesem Inhalt:
+
+```bash
+conf:phpfpm:tcp:php74:9000  
+```
+
+Starte devilbox wie folgt:
+
+```bash
+docker-compose up php httpd bind php74
+```
+
+{{% notice note %}}
+Ein Beispiel:
+<table>
+  <tr>
+    <th>Projekt</th>
+    <th>.env php-Version</th>
+    <th>backend.cfg vorhanden?</th>
+    <th>bind Option</th>
+    <th>Projekt läuft unter<th>
+  </tr>
+  <tr>
+    <td>contao5</td>
+    <td>8.2</td>
+    <td>Nein</td>
+    <td>-</td>
+    <td>8.2</td>
+  </tr>
+  <tr>
+    <td>contao4</td>
+    <td>8.2</td>
+    <td>Ja</td>
+    <td>php74</td>
+    <td>7.4</td>
+  </tr>
+</table>
+
+{{% notice tip %}}
+Du kannst in jedem Projektverzeichnis prophylaktisch die backend.cfg anlegen und den Inhalt auskommentieren, wenn das Projekt nicht gesondert behandelt werden soll. 
+{{% /notice %}}
+
+{{% notice tip %}}
+Im devilbox Dashboard siehst du unter `Virtual Hosts` in der Spalte `Backend', welche php-Version tatsächlich für ein Projekt im Einsatz ist. (Noch gibt es dort einen kleinen Fehler: Auskommentierte Konfigurationen werden nicht als passiv erkannt.)
+{{% /notice %}}
+
+
+## Nützliche Informationen: PHP Xdebug
+
+Im [Devilbox-Handbuch](https://devilbox.readthedocs.io/en/latest/intermediate/configure-php-xdebug.html?highlight=xdebug#configure-php-xdebug-1) findest du eine nicht ganz aktuelle Dokumentation, die aber das grundsätzliche Vorgehen gut aufzeigt.
+
+Für Vscode funktioniert diese Konfiguration sehr gut:
```

Comment:
```suggestion
In meiner IDE funktioniert folgende Konfiguration sehr gut:
```

Reply:
I am not sure if it then becomes clear that this configuration is for Vscode only.

Reply:
The configuration should not have anything to do with an IDE. We don't want to force people to use Vscode or any other IDE. They should be able to use Devilbox with their IDE.

Reply:
Also dann nehmen wir deinen Vorschlag. Wenn Nutzer*innen einer anderen IDE abweichende Konfis haben sollten, kann das ja hier noch ergänzt werden.